dashbuilder-deps/pom.xml

```diff
@@ -21,7 +21,7 @@
   <properties>
     <!-- Keep in sync with parent's <version> in dashbuilder-parent-metadata -->
     <version.org.jboss.integration-platform>6.0.8.Final</version.org.jboss.integration-platform>
-    <version.org.uberfire>0.9.0-SNAPSHOT</version.org.uberfire>
+    <version.org.uberfire>0.9.1-SNAPSHOT</version.org.uberfire>
```

Comment:
@mbiarnes Hi, is this correct? I could not get everything working without upping this.

Reply:
I approved this - at least the update of version - this is correct for sure